tests/test_app_metadata.py

```diff
@@ -55,6 +55,18 @@
                 "OpenJDK Runtime Environment (build 1.8.0_275-b01)\n"
                 "OpenJDK 64-Bit Server VM (build 25.275-b01, mixed mode)",
                 "libjvm_elfid": "buildid:0542486ff00153ca0bcf9f2daea9a36c428d6cde",
+                "jvm_flags": [
```

Comment:
Let's add a test for JDK 9+, afaict the output of `jcmd VM.flags -all` is a bit different there.
In addition, consider adding unit tests for the parsing alone.

Reply:
Are we adding unit tests for the parsing?

Reply:
no we are not adding unitests